testing/test_webinterface.py

```diff
@@ -84,7 +86,7 @@ def test_webinterface_with_activate(webinterface_service):
         in app_log
     )
 
-    assert "Deactivating setup id=1" in app_log
+    assert "Deactivating setup id=" in app_log
```

Comment:
Why was this `id` lost?

Reply:
Not lost. Just counted up in DB. Was to lazy to clean DB every run. Can be 0 if DB is new.